homeassistant/components/daikin_madoka/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "daikin_madoka",
+  "name": "Daikin Madoka",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/daikin",
```

Comment:
This is not the documentation for the correct domain

Reply:
Lol.. copy & paste. Updated.